<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>671</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_log">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LOG</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FITLOG DASHBOARD</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Output calories :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="lcdNumber_intake"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Intake calories :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber_output"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_fullname">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Goal Calories :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_history">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VIEW HISTORY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_logout">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LOG OUT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_refresh">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REFRESH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stutus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Prompt</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DD/MM/YYYY</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
